message_definitions/v1.0/common.xml

```diff
@@ -1471,6 +1474,16 @@
         <param index="6">Empty</param>
         <param index="7">Empty</param>
       </entry>
+      <entry value="198" name="MAV_CMD_DO_SET_ROI_SYSID">
+        <description>Follow vehicle by system ID.</description>
+        <param index="1" label="System ID" minValue="0" maxValue="255" increment="1">sysid</param>
```

Comment:
minValue should be 1 because 0 is an invalid  address for a vehicle (broadcast address).

Reply:
Yep.  I was thinking zero might be "first system ID seen", but I'm not doing that as part of this PR.

Changing.